react/features/whiteboard/functions.ts

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable import/order */
+import md5 from 'js-md5';
+
+// @ts-ignore
+import { getPinnedParticipant } from '../../features/base/participants';
+import { appendURLParam } from '../base/util/uri';
+
+// @ts-ignore
+import { getCurrentRoomId, isInBreakoutRoom } from '../breakout-rooms/functions';
+
+const getWhiteboardState = (state: any): any => state['features/whiteboard'];
+
+/**
+ * Indicates whether the whiteboard is open.
+ *
+ * @param {Object} state - The state from the Redux store.
+ * @returns {boolean}
+ */
+export const isWhiteboardOpen = (state: any): boolean => getWhiteboardState(state).enabled;
```

Comment:
The function is called "open" and yet it checks for some "enabled" flag. That sounds somewhat wrong.

Reply:
I did some last minute changes to incorporate the config enabled flag. I agree, I'll fix the naming.